.github/workflows/publish-release.yaml

```diff
@@ -0,0 +1,111 @@
+name: Publish release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    # Note: `prepare-release.yaml` sets this commit message
+    if: ${{ contains(github.event.head_commit.message, 'release @tutorialkit packages') }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup-and-build
+
+      # Sets steps.resolve-release-version.outputs.version
+      - uses: ./.github/actions/resolve-release-version
+        id: resolve-release-version
+
+      - name: Publish to npm
+        run: >
+          pnpm --recursive
+          --filter "@tutorialkit/*"
+          --filter create-tutorial
+          exec pnpm publish --provenance
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
We'll need to manually add this secret to the project.

Reply:
@d3lm I think if you can add @stackblitz-gitops to the packages, then we should be able to reuse the token of @stackblitz-gitops.

@apai4 Do you think that would be fine? Or should we have a different token?

Reply:
I pinged Pai about this because I cannot add NPM access tokens since I don't have permissions to generate them.

Reply:
I'll create a new token for `stackblitz-devops` on NPM for this repo and add it to `secrets.NPM_TOKEN`. `stackblitz-devops` will need write access to the NPM package.

Reply:
The token has been created and added to the repo. Let me know if it works once `stackblitz-gitops` is granted access

Reply:
@d3lm can you grant access to `stackblitz-gitops` on all the packages?

Reply:
Yes, I can do that!

Reply:
Invite is out. I think @apai4 needs to accept it.

Reply:
@d3lm do you have permissions to check if this is completed now? Publish step failed at https://github.com/stackblitz/tutorialkit/actions/runs/9791481416/job/27035364611